src/server/scripts/Spells/spell_rogue.cpp

```diff
@@ -162,6 +165,36 @@ class spell_rog_blade_flurry : public AuraScript
     Unit* _procTarget = nullptr;
 };
 
+// 31230 - Cheat Death
+class spell_rog_cheat_death : public AuraScript
+{
+    bool Validate(SpellInfo const* spellInfo) override
+    {
+        return ValidateSpellInfo({ SPELL_ROGUE_CHEAT_DEATH_DUMMY, SPELL_ROGUE_CHEATED_DEATH, SPELL_ROGUE_CHEATING_DEATH })
+            && ValidateSpellEffect({ { spellInfo->Id, EFFECT_1 } });
+    }
+
+    void HandleAbsorb(AuraEffect* /*aurEff*/, DamageInfo& dmgInfo, uint32& absorbAmount)
```

Comment:
You need to set the absorbAmount to 0 if you dont want to absorb anything, because right now you're absorbing the amount that is stored in the basepoints

Reply:
the sniff shows spell absorb

Reply:
> the sniff shows spell absorb

how much? even if you have Cheated Death active?

Reply:
yes, first you cast spells then absorb log (on sniff)

Reply:
> yes, first you cast spells then absorb log (on sniff)

How much do you absorb if cheat death is on CD?

Reply:
if cheat death is on cd you die

Reply:
If that’s the case, read my first comment again